feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -49,6 +49,17 @@ func EqualToDefault[T any](query ygnmi.SingletonQuery[T], val T) check.Validator
 	return check.Equal(query, val)
 }
 
+// CheckPresence check for the leaf presense only when MissingValueForDefaults
+// deviation is marked false.
+func CheckPresence[T any](query ygnmi.SingletonQuery[T]) check.Validator {
```

Comment:
Do you mind explaining the benefits of using a generic function for this?

Reply:
@aaronmillisor Initially this function was made to check for `DatabaseOverloads` which carries value of `uint32` type. But that makes this function very specific for this leaf only. I think, better approach is to keep the function generation for all leaf, just like `EqualToDefault` is generic.